assets/components/pages/event-detail/event-detail.twig

```diff
@@ -1,116 +1,162 @@
-{% set isHomepage = true %}
+{% set isSpecial = true %}
 
 {% extends "@templates/base/base.twig" %}
 
 {% block header %}
   {% include '@organisms/header/header-light.twig' with { site_title: 'Event title' } %}
 {% endblock %}
 
+{% block breadcrumb %}
+<div class="breadcrumb-container">
+  {% include '@molecules/breadcrumb/breadcrumb-tagged.twig' %}
+</div>
+{% endblock %}
+
 {% block content %}
 
-<div class="container-full bg-gray-100 py-5 mb-5">
-  <div class="container">
-    <div class="row">
-      <div class="col-lg-6">
-        <p class="lead">Summary of the event. For example :</p>
-        <p>L'exposition Kung Fu Motion retrace l’histoire dynamique des arts martiaux traditionnels de la Chine méridionale et de Hong Kong, à travers le prisme de technologies d’archivage de pointe. Située à l’intersection des arts, de la science et des pratiques rituelles, cette exposition expérimentale ouvre de nouvelles perspectives sur notre rapport au patrimoine culturel immatériel.</p>
+<div class="event-wrapper container-full" itemscope itemtype="http://schema.org/Event">
+  <div class="container-full py-5">
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-12">
+          <h1 class="event-title" itemprop="name">Event title may contains several words on maybe multiple lines, who knows?</h1>
+        </div>
       </div>
-      <div class="col-lg-6">
-        {% include '@atoms/picture/picture.twig' %}
+      <div class="row py-3">
+        <div class="col-lg-8 col-md-6">
+          {% include '@atoms/picture/picture.twig' %}
+        </div>
+        <div class="col-lg-4 col-md-6 bg-gray-100">
```

Comment:
This look like a new component to me, can you extract it from this example page and define it as a properly documented component ?

Reply:
Right, I'm adding an issue to address this #270